features/support/env.rb

```diff
@@ -1,5 +1,35 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__) + '/../../lib')
-require 'aruba/cucumber'
+
+# Has to be the first file required so that all other files show coverage information
+require 'simplecov'
+
+#
+# Standard Library
+#
+
 require 'fileutils'
+require 'pathname'
+
+require 'aruba/cucumber'
 require 'rspec/expectations'
 
+Before do |scenario|
```

Comment:
I only meant to `simplecov_setup` to be loaded in the child process, hence why it's path is used in RUBYOPT on line 34.  I don't want it loaded in the main cucumber process.